pyrsia_node/src/main.rs

```diff
@@ -164,7 +180,8 @@ async fn main() {
                 .or(v2_blobs)
                 .or(v2_blobs_post)
                 .or(v2_blobs_patch)
-                .or(v2_blobs_put),
+                .or(v2_blobs_put)
+                .or(v2_peers),
```

Comment:
Though we should version our own api I dont think we are at v2 😆 

```suggestion
                .or(peers),
```

Reply:
yes as mentioned before will be replaced in #112